alphadia/fdrx/base.py

```diff
@@ -0,0 +1,137 @@
+# make it an abc
+import abc
+import sklearn.base
+from typing import List
+import pandas as pd
+import numpy as np
+from alphadia.fragcomp import FragmentCompetition
+from alphadia.fdrx.stats import _fdr_to_q_values, _get_pep, _get_q_values, _keep_best
+from alphadia.fdrx.plotting import (
+    _plot_fdr_curve,
+    _plot_roc_curve,
+    _plot_score_distribution,
+)
+
```

Comment:
What is your definition of a private function? One could argue that if you need to import these functions here, they should not be private anymore.

Reply:
Good point, this is really soft. The idea is that these functions should not be used outside the python module and there can be breaking changes any time.